lib/plugins/package/lib/packageService.js

```diff
@@ -44,10 +44,13 @@ module.exports = {
 
     if (this.serverless.service.package.individually) {
       const allFunctions = this.serverless.service.getAllFunctions();
-      const packagePromises = _.map(allFunctions, functionName =>
-        this.packageFunction(functionName));
+      const concurrency = this.serverless.service.package.sequentially ? 1 : Infinity;
```

Comment:
Not sure if that useful but maybe we can extend it so that one can specify this option.

Variation: "I don't care"
```yml
package:
  sequentially: true
```

Variation: "Performance tweakers"
```yml
package:
  sequentially: 5
```